<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="start_x" default="5.98928"/>
  <arg name="start_y" default="-1.60184"/>
  <arg name="start_yaw" default="-0.0570243"/>

  <arg name="goal_x" default="4.02556"/>
  <arg name="goal_y" default="-1.004421"/>
  <arg name="goal_yaw" default="0.06504"/>

  <group>
    <push-ros-namespace namespace="vox_nav_planner_server"/>
    <node pkg="vox_nav_planning" exec="box2d_planner_client" name="vox_nav_box2d_planner_client_rclcpp_node" output="screen">
      <!-- start pose -->
      <param name="start_x" value="$(var start_x)"/>
      <param name="start_y" value="$(var start_y)"/>
      <param name="start_yaw" value="$(var start_yaw)"/>

      <!-- goal pose -->
      <param name="goal_x" value="$(var goal_x)"/>
      <param name="goal_y" value="$(var goal_y)"/>
      <param name="goal_yaw" value="$(var goal_yaw)"/>
    </node>
  </group>
</launch>
